graphwalker-core/src/main/java/org/graphwalker/core/model/Edge.java

```diff
@@ -208,6 +209,34 @@ protected RuntimeEdge createCache() {
   }
 
   /**
+   * Gets the dependency of the edge.
+   *
+   * @return The dependency as double.
+   * @see Edge#setDependenct
+   */
+  public Double getDependency() {
+	return dependency;
+  }
+
+  /**
+   * The dependency shows how much targetVertex depends on sourceVertex.
+   * One way to obtain the dependency is by using
+   *  process mining to generate a model out of log files.
+   *
+   * @param dependency a double between 0 and 1
+   * @return The edge
+   */
+  public Edge setDependency(Double dependency) {
+	if (dependency > 1 && dependency <= 100) {
```

Comment:
This would lead to problems with doubles between 0 and 1. Someone using the dependency value 1, could lead to some unexpected results, right?

Would it not be better to define what the value in the json represents?
I suggest that it should always be a percentage value, expected to be equal or larger than 0, and, equal or less than 100, or non-existent.